awesome.c

```diff
@@ -514,6 +512,12 @@ main(int argc, char **argv)
             else
                 fatal("-c option requires a file name");
             break;
+          case 's':
+            if(a_strlen(optarg))
+                string_array_append(&searchpath, a_strdup(optarg));
+            else
+                fatal("-s option requires a directory name");
+            break;
```

Comment:
Whoops, yeah. However, this whole error should just be removed instead:
https://github.com/awesomeWM/awesome/pull/1063
https://github.com/awesomeWM/awesome/pull/1063/commits/4cbf87b441bf36961827a7372fcf8172bc03a75a